dg/config/cluster_credentials_manager.py

```diff
@@ -204,6 +207,36 @@ def get_current_cluster(self) -> Union[AbstractCluster, None]:
 
         return cluster
 
+    def get_current_credentials(self) -> ContextData:
+        """Returns user and current cluster credentials
+
+        Returns
+        -------
+        ContextData
+            user and current cluster credentials
+        """
+
+        dg_credentials_file_path = (
+            data_gate_configuration_manager.get_dg_credentials_file_path()
+        )
+        result: ContextData
+
+        if dg_credentials_file_path.exists() and (
+            dg_credentials_file_path.stat().st_size != 0
+        ):
+            with open(dg_credentials_file_path) as json_file:
+                result = json.load(json_file)
+        else:
+            result = {}
+
+        current_cluster = self.get_current_cluster()
+
+        if current_cluster is not None:
```

Comment:
Not directly related to this PR, but it would probably be advantageous to have get_current_cluster() throw an exception instead of returning `None` in these casese, this way all get_current_cluster() calls can be accessed in a reliable way using try-catch, enforcing a clean control flow and avoiding unexpected None  return situations.

Reply:
The semantics of `None` is that no cluster is selected, which is a valid state and therefore no exception must be thrown.